.eslintrc

```diff
@@ -0,0 +1,7 @@
+{
+  "extends": ["eslint-config-axiom"],
```

Comment:
While you're here, might as well use the new package `@brandwatch/eslint-config-axiom`.